### PR TITLE
[docs] Rewiewed docs StorageClass for cloud-providers

### PR DESCRIPTION
**Pull request by mortelumina98**
## Description
Rewiewed docs StorageClass for cloud-providers.
Changes have been made in the [PR](https://github.com/deckhouse/deckhouse/pull/9591).

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Rewiewed docs StorageClass for cloud-providers.
impact_level: low
```

### CODE REVIEW

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 1]
```diff
@@ -10,11 +10,14 @@ properties:
...
-          Иначе будет использован первый StorageClass из создаваемых модулем cloud-provider (если какой-либо модуль cloud-provider-а используется).
-        * Воспользуйтесь этим параметром, чтобы изменить StorageClass по умолчанию в кластере, поскольку вручную добавлять default-аннотации в StorageClass-ы запрещено.
+    description: |-
+      Имя StorageClass, который будет использоваться по умолчанию в кластере.
+      Если значение не указано, фактический StorageClass по умолчанию будет определен следующим образом:
+
+       * Произвольный StorageClass в кластере, имеющий default-аннотацию; 
+       * Первый (по алфавиту) StorageClass из создаваемых модулем облачного провайдера (если какой-либо модуль облачного провайдера используется). 
+        
+      Воспользуйтесь этим параметром, чтобы изменить StorageClass по умолчанию в кластере, поскольку вручную добавлять default-аннотации в StorageClass запрещено.
```
Comment (z9r5) [inline global-hooks/openapi/doc-ru-config-values.yaml:20]:
```suggestion
    description: |-
      Имя StorageClass, который DKP определит как StorageClass, используемый в кластере по умолчанию.
      
      DKP запрещает вручную добавлять или изменять default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`) у StorageClass. Используйте параметр, чтобы определять StorageClass, используемый в кластере по умолчанию.

      Если значение не указано, фактический StorageClass по умолчанию будет определяться в следующем порядке:

       * StorageClass в кластере, имеющий default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`); 
       * Первый (по алфавиту) StorageClass из создаваемых модулем облачного провайдера (если какой-либо модуль облачного провайдера используется). 
```

File: global-hooks/openapi/config-values.yaml [hunk 2]
```diff
@@ -14,11 +14,14 @@ properties:
   defaultClusterStorageClass:
     type: string
     default: null
-    description: |
+    description: |-
       The name of the StorageClass that will be annotated as default in the cluster.
```
Comment (z9r5) [inline global-hooks/openapi/config-values.yaml:18]:
```suggestion
      The name of the StorageClass that DKP will define as the default StorageClass used in the cluster.
```

File: global-hooks/openapi/config-values.yaml [hunk 3]
```diff
@@ -14,11 +14,14 @@ properties:
...
-        * If the value is omitted, the current default StorageClass will be kept, if it exists in the cluster.
-          Otherwise, the first StorageClass created by the cloud-provider module will be used (if any cloud-provider module is used).
-        * Use this parameter to change the default StorageClass in the cluster, since manually adding default-annotations to StorageClasses is prohibited.
+      If the parameter is omitted, the default StorageClass is either:
+
+       * an arbitrary StorageClass present in the cluster that has the default annotation;
+       * The first (alphabetically) StorageClass created by the cloud provider module (if any cloud provider module is used).
+
+      Use this parameter to change the default StorageClass in the cluster, as manually adding the default annotation to a StorageClass is not allowed.
```
Comment (z9r5) [inline global-hooks/openapi/config-values.yaml:24]:
```suggestion

      DKP prohibits manually adding or modifying the default annotation (`storageclass.kubernetes.io/is-default-class='true'`) for StorageClass. Use the parameter to define the StorageClass that is used as the default in the cluster.

      If the parameter is omitted, the default StorageClass is either:

       * A StorageClass in the cluster that has the default annotation (`storageclass.kubernetes.io/is-default-class='true'`);
       * The first (alphabetically) StorageClass created by the cloud provider module (if any cloud provider module is used).

      Use this parameter to change the default StorageClass in the cluster, as manually adding the default annotation to a StorageClass is not allowed. 
```

**Review by z9r5 — APPROVED** [review]

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 4]
```diff
@@ -10,11 +10,15 @@ properties:
...
     default: null
-    description: |
-      Имя StorageClass, который будет назначен по умолчанию в кластере.
-        * Если значение отсутствует, то будет сохранен текущий StorageClass по умолчанию, при его наличии в кластере.
-          Иначе будет использован первый StorageClass из создаваемых модулем cloud-provider (если какой-либо модуль cloud-provider-а используется).
-        * Воспользуйтесь этим параметром, чтобы изменить StorageClass по умолчанию в кластере, поскольку вручную добавлять default-аннотации в StorageClass-ы запрещено.
+    description: |-
+      Имя StorageClass, который DKP определит как StorageClass, используемый в кластере по умолчанию.
+      
+      DKP запрещает вручную добавлять или изменять default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`) у StorageClass. Используйте параметр, чтобы определять StorageClass, используемый в кластере по умолчанию.
```
Comment (z9r5) [inline global-hooks/openapi/doc-ru-config-values.yaml:16]:
```suggestion
      DKP запрещает вручную добавлять или изменять default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`) у StorageClass. Используйте параметр, чтобы определять или изменять StorageClass, используемый в кластере по умолчанию.
```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 5]
```diff
@@ -10,11 +10,15 @@ properties:
...
     default: null
-    description: |
-      Имя StorageClass, который будет назначен по умолчанию в кластере.
-        * Если значение отсутствует, то будет сохранен текущий StorageClass по умолчанию, при его наличии в кластере.
-          Иначе будет использован первый StorageClass из создаваемых модулем cloud-provider (если какой-либо модуль cloud-provider-а используется).
-        * Воспользуйтесь этим параметром, чтобы изменить StorageClass по умолчанию в кластере, поскольку вручную добавлять default-аннотации в StorageClass-ы запрещено.
+    description: |-
+      Имя StorageClass, который DKP определит как StorageClass, используемый в кластере по умолчанию.
+      
+      DKP запрещает вручную добавлять или изменять default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`) у StorageClass. Используйте параметр, чтобы определять или изменять StorageClass, используемый в кластере по умолчанию.
```
Comment (z9r5) [inline global-hooks/openapi/doc-ru-config-values.yaml:16]:
```suggestion
      DKP запрещает вручную добавлять или изменять default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`) у StorageClass. Используйте параметр, чтобы определять или изменять StorageClass используемый в кластере по умолчанию.
```

File: global-hooks/openapi/doc-ru-config-values.yaml [hunk 6]
```diff
@@ -10,11 +10,15 @@ properties:
...
-      Имя StorageClass, который будет назначен по умолчанию в кластере.
-        * Если значение отсутствует, то будет сохранен текущий StorageClass по умолчанию, при его наличии в кластере.
-          Иначе будет использован первый StorageClass из создаваемых модулем cloud-provider (если какой-либо модуль cloud-provider-а используется).
-        * Воспользуйтесь этим параметром, чтобы изменить StorageClass по умолчанию в кластере, поскольку вручную добавлять default-аннотации в StorageClass-ы запрещено.
+    description: |-
+      Имя StorageClass, который DKP определит как StorageClass используемый в кластере по умолчанию.
+      
+      DKP запрещает вручную добавлять или изменять default-аннотацию (`storageclass.kubernetes.io/is-default-class='true'`) у StorageClass. Используйте параметр, чтобы определять или изменять StorageClass используемый в кластере по умолчанию.
+
+      Если значение не указано, фактический StorageClass по умолчанию будет определяться в следующем порядке:
```
Comment (z9r5) [inline global-hooks/openapi/doc-ru-config-values.yaml:18]:
```suggestion
      Если значение не указано, фактический StorageClass используемый в кластере по умолчанию будет определяться в следующем порядке:
```

**Comment by deckhouse-BOaTswain** [comment]
Backport failed. See [Job](https://github.com/deckhouse/deckhouse/actions/runs/11852798571) for details.

**Comment by z9r5** [comment]
Cherry-picked into release-1.66

**Pull request merged**